README.md

```diff
@@ -31,6 +46,13 @@ VSCode needs .json, not .yaml -> use `npm run build` to generate .json from .yam
    which is at the end, applying correct scopes.
 7. Repeat step 4.
 
+For features implemented in typescript, the entry point for the project is `src/extension.ts`.
+Use `npm run compile-ts` to compile the typescript code, or `npm run watch` to 
+automatically recompile on changes.
+
+Use `npm run compile` to compile typescript code and convert the TextMate grammar
```

Comment:
Aha, so `npm run compile` does everything, while `compile-ts` and `compile-yaml` do specific stuff. Ok makes sense!